### DISCUSSION TITLE
Is there a way with HTML or Markdown to paste formatted field codes into Microsoft word?

### DISCUSSION THREAD

**Discussion by ZerkerEOD** [espanso/espanso#1557] [Q&A]
I am trying to find a way to template stuff that I write into Microsoft word and we use custom field codes (not that figured it out with normal ones either). Say something like a client name that is unique in that document, but I want my Espansa to expand the field code. 

Currently I have a macro built on my Stream Deck that does the custom CTRL+F9 then types in the field codes content then update it to pull the filed code for the saved information in the document. 

I was hoping with HTML or Markdown there would be a way to paste formatted text in that would be the field code so then when we do a full f9 update on everything it pulls the saved document information.

**Comment by reitzig** [comment]
Indeed, something like

```yaml
- trigger: ":ref"
  markdown: "[label](url)"
```

works in most "rich"-text editors for me.

**Reply by ZerkerEOD** [reply]
I got that when linking websites, but I'm talking about field codes inside a document. So we'll have a field codes document property that we store stuff like testing start and stop dates, client name and a few others. I'm trying to get it to reference those so that when we update the document it will say the clients name or date rather than someone I need to manually convert after.

**Comment by ZerkerEOD** [comment]
As an update, the field I am talking about looks like this when you turn on to review what is in fields 

`{ DOCPROPERTY Client \* MERGEFORMAT }`

But you can't just type that in and word automatically convert it. Word will just see it because it is unformatted text which is why I thought pasting in HTML or markdown formated text could fix it but I don't think markdown can do it, and I think I would need to do a script that presses keys and formats it.

**Comment by smeech** [comment]
Duplicate of #1162?
